types/react-relay/test/react-relay-tests.tsx

```diff
@@ -501,3 +502,10 @@ requestSubscription(
         },
     }
 );
+
+// ~~~~~~~~~~~~~~~~~~~~~
+// Context
+// ~~~~~~~~~~~~~~~~~~~~~
+
+ReactRelayContext.Consumer.prototype;
+ReactRelayContext.Provider.prototype;
```

Comment:
I actually meant a snippet of actual API usage, but at a glance I only see `createFragmentSpecResolver` as a potential option and it doesn’t seem trivial to use that right now, so let’s just go with this for now 👍

Reply:
yeah. I mean, I didn't actually specify the Context type or any other type in this PR, that already exists and is hopefully tested in relay-runtime. i just added a `const` in the module, so this tests that the `const` is indeed there 😅

Reply:
Right right, I just try to identify opportunities for backfilling some of the debt we have as well ;)